robots/preservation/create_moab.rb

```diff
@@ -0,0 +1,26 @@
+
+module Preservation
+  # Update or create moab object
+  class CreateMoab < Base
+    ROBOT_NAME = 'creaet-moab'.freeze
+
+    def initialize(opts={})
+      super(REPOSITORY, WORKFLOW_NAME, ROBOT_NAME, opts)
+    end
+
+    def perform(druid)
+      storage_object = Moab::StorageServices.find_storage_object(druid, true)
+      storage_object.object_pathname.mkpath
+      create_moab(storage_object)
+    end
+
+    def create_moab(storage_object)
```

Comment:
this is more aptly named "create_version", don't you think?  or "update_moab"?   since it may be the Moab's first version, or it may be a new version